test/acceptance/bytegroup/go.mod

```diff
@@ -0,0 +1,11 @@
+module bytegroup
+
+go 1.13
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.1.0
+	github.com/Azure/go-autorest/autorest v0.9.3
```

Comment:
I'm guessing there's no real dependency on go-autorest?  You can run `go mod tidy` to clean this stuff up.

Reply:
Done